scss/_base_forms-range.scss

```diff
@@ -15,7 +15,7 @@
     // stylelint-enable property-no-vendor-prefix
     background-color: $color-transparent;
     border-radius: $track-radius;
-    margin: $sp-small 0;
+    margin: ($sp-small - $track-height * 0.5) 0;
```

Comment:
This is using a legacy spacing variable. Should be replaced with new dedicated to vertical spacing `$spv--small` now that we update it. Thanks!

Reply:
It's fixed now!